Source/MQTTnet/Client/MqttClientOptionsBuilder.cs

```diff
@@ -86,33 +112,69 @@ public MqttClientOptionsBuilder WithWebSocketServer(string uri)
             return this;
         }
 
+
```

Comment:
I already started to create a custom object for the parameters to avoid lots of parameters. My idea is to use it like this.
```
.WithTls(new WithTlsParameters{
    SslProtocol = Ssl3,
    CertificateValidationCallback = <whatever>,
    Certificates = LoadFromSomewhere()
});
````

All properties will have proper default values. 
I believe this is the right time to implement this because the large amout of overloads is a mess.

What do you think?

Reply:
I agree it's more clean this way.